docs/UserGuide.md

```diff
@@ -425,18 +427,21 @@ EmployeeManager data are saved in the hard disk automatically after any command
 
 ### Editing the data file
 
-EmployeeManager data are saved automatically as a JSON file `[JAR file location]/data/employeemanager.json`. Advanced users are welcome to update data directly by editing that data file.
+EmployeeManager automatically saves your data as a JSON file `[JAR file location]/data/employeemanager.json`. 
+For advanced users, you have the option to directly modify this data file to update records. After editing, simply `import` the
```

Comment:
I thought you just need to restart the application to view the updated changes? I don't think there is a need to `import` the file.